subprojects/language-groovy/src/integTest/groovy/org/gradle/groovy/GroovyDocGenericsIntegrationTest.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.groovy
+
+import org.gradle.integtests.fixtures.TargetCoverage
+import org.gradle.testing.fixture.GroovydocCoverage
+import spock.lang.Issue
+
+@TargetCoverage({ GroovydocCoverage.GENERICS_ERROR_COVERAGE })
+class GroovyDocGenericsIntegrationTest {
+    @Issue(["GRADLE-3174", "GRADLE-3463"])
+    def "Error in Groovydoc generation is logged"() {
```

Comment:
Well spotted. I've changed the test now to look for the output of `[ant:groovydoc]` instead.